script.js

```diff
@@ -1,65 +1,86 @@
-const names = document.getElementById('title');
-const pass = document.getElementById('author');
+/* eslint-disable max-classes-per-file */
+const title = document.getElementById('title');
+const author = document.getElementById('author');
 const button = document.getElementById('button');
-const list = document.getElementById('list');
 
-list.style.width = '20%';
-
-function Books(title, author) {
-  this.title = title;
-  this.author = author;
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
 
-function setData() {
-  const toObject = JSON.parse(localStorage.getItem('books-list'));
-  return toObject;
-}
+class Storage {
+  static getBooks() {
+    let booksArray;
+    if (localStorage.getItem('books') === null) {
+      booksArray = [];
+    } else {
+      booksArray = JSON.parse(localStorage.getItem('books'));
+    }
+    return booksArray;
+  }
 
-let booksArray = setData() ?? [];
+  static addBookStorage(book) {
+    const booksArray = Storage.getBooks();
+    booksArray.push(book);
```

Comment:
## OPTIONAL
- [ ] Please make sure to use `concat` method instead of `push` because push mutates the array;

- The `push()` method is a generic method similar to `call()` or `apply()`. It will mutate your array (or object), pushing a new value into it

- The `concat()` method returns a new array, with the values merged. This also avoids mutation side effects.